src/main/java/Duke.java

```diff
@@ -1,10 +1,143 @@
+                System.out.println("Nice! I've marked this task as done:\n" +
+                        "[" + currTask.getStatusIcon() + "] " + currTask.description);
+                str = sc.nextLine();
+                str = sc.nextLine();
+            }
+
+            else if (str.equals("unmark")) {
+                int num = sc.nextInt();
+                Task currTask = taskArray.get(num - 1);
+                currTask.unMark();
+                System.out.println("OK, I've marked this task as not done yet:\n" +
+                        "[" + currTask.getStatusIcon() + "] " + currTask.description);
+                str = sc.nextLine();
+                str = sc.nextLine();
+            }
+            else if (str.equals("todo")) {
+                String input = sc.nextLine();
+                checkInput(input);
+                taskArray.add(new ToDo(input));
+                System.out.println("Got it. I've added this task:\n" + "  " +
+                        taskArray.get(counter));
+                counter++;
+                System.out.println("Now you have " + counter + " tasks in the list.");
+                str = sc.nextLine();
+            }
+            else if (str.equals("deadline")) {
+                String input = sc.nextLine();
+                checkInput(input);
+                String by = sc.nextLine();
+                checkInput(by);
+                taskArray.add(new Deadline(input, by));
+                System.out.println("Got it. I've added this task:\n" + "  " +
+                        taskArray.get(counter));
+                counter++;
+                System.out.println("Now you have " + counter + " tasks in the list.");
+                str = sc.nextLine();
+            }
+            else if (str.equals("event")) {
+                String input = sc.nextLine();
+                checkInput(input);
+                String period = sc.nextLine();
+                checkInput(period);
+                taskArray.add(new Events(input, period));
+                System.out.println("Got it. I've added this task:\n" + "  " +
+                        taskArray.get(counter));
+                counter++;
+                System.out.println("Now you have " + counter + " tasks in the list.");
+                str = sc.nextLine();
+            }
+            else if (str.equals("delete")) {
+                int num = sc.nextInt() - 1;
+                Task temp = taskArray.get(num);
+                taskArray.remove(num);
+                System.out.println("Noted. I've removed this task:\n" + "  " +
+                        temp.toString());
+                counter--;
+                System.out.println("Now you have " + counter + " tasks in the list.");
+                str = sc.nextLine();
+
+            }
+
+            else {
+                taskArray.add(new Task(str));
+                counter++;
+                System.out.println("added: " + str);
+                str = sc.nextLine();
+            }
+
```

Comment:
Some whitespace is good, but try to avoid too many empty lines. I find it hard to follow.